### PR TITLE
Fixing tests

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java [hunk 1]
```diff
@@ -117,24 +118,23 @@ private void commaSeparatedEntryTransformer(String commaSeparatedEntryKey,
...
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.APPLICANT_EMAIL, "applicantEmail");
         return exceptionRecordToCcdFieldsMap;
     }
 
-    private void applyMappingsForAddress(String prefix, String unusualField, List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
+    private void applyMappingsForAddress(String prefix, String parentField, List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
         String nestedFieldPrefix = StringUtils.capitalize(prefix + "Address");
         addMappingsTo(
-            unusualField,
+            parentField,
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]: :+1:

Comment (yarpo) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]: who has used such a useless name!?

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java [hunk 2]
```diff
@@ -177,40 +195,6 @@ public void shouldTransformAddresses() {
         );
     }
 
-    @Test
-    public void shouldMapPhoneNumber() {
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java]: why have you removed these?

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java (hunk 1 above)
Comment (jakub-kozlowski) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java]: they were moved to `shouldPassValidationForValidMandatoryAndOptionalFields`

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java (hunk 2 above)
Comment (jakub-kozlowski) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java]: sorry, pasted to the wrong window 🙈  
they were moved to shouldPassValidationForValidMandatoryAndOptionalFields

**Pull request merged**